packages/fluentui/react-bindings/src/hooks/useStyles.ts

```diff
@@ -63,7 +63,7 @@ export const useStyles = <StyleProps extends PrimitiveProps>(
     className = process.env.NODE_ENV === 'production' ? '' : 'no-classname-🙉',
     composeOptions,
     mapPropsToStyles = () => ({} as StyleProps),
-    mapPropsToInlineStyles = () => ({} as InlineStyleProps<StyleProps>),
+    mapPropsToInlineStyles = () => ({} as InlineStyleProps<any>),
```

Comment:
why does it make no sense ? you could make `mapPropsToInlineStyles` take a generic too

Reply:
It will require to modify all existing components as `useStyles()` will have to accept a second parameter in the generic, it will be too much for this PR. At the same time it will affect only `styles` property in `InlineStyleProps` type and it will not improve type safety compared to required effort 😥